### ISSUE TITLE
Possibility to choose a custom folder for storage of recordings and texts

### ISSUE THREAD

**Issue by Warden20** [maxkrieger/voiceliner#111]
It would be great to be able to choose your own folder to store recordings and transcripts of recordings.
This would simplify the process of exporting and importing to other programs and facilitate backup options with other applications such as Syncthing.

**Comment by metafarion** [comment]
Yes, this would be fantastic. No need to export each individual item to get it into the storage of your choice, just have the program save them there from the start.
